chatterbot/tagging.py

```diff
@@ -23,7 +23,10 @@ def __init__(self, language=None):
 
         self.punctuation_table = str.maketrans(dict.fromkeys(string.punctuation))
 
-        self.nlp = spacy.load(self.language.ISO_639_1.lower())
+        if self.language.ISO_639_1.lower() == 'en':
+            self.nlp = spacy.load('en_core_web_trf')
+        else:
+            self.nlp = spacy.load(self.language.ISO_639_1.lower())
```

Comment:
I've made a different change as a part of #2389 that should achieve a similar effect with a bit more flexibility for other languages: https://github.com/gunthercox/ChatterBot/pull/2389/files#diff-5fad1f9c3be573497a8f25e97994e2475e240f7034eb59a04d6cec8137efbbd9R156-R163